babel.config.js

```diff
@@ -1,80 +0,0 @@
-module.exports = (api) => {
-  const env = api.env();
-
-  const reactOptions = {
-    development: false,
-    runtime: 'automatic',
-  };
-
-  const envOptions = {
-    useBuiltIns: "usage",
-    corejs: { version: "3.30" },
-    debug: false,
-    include: [
-      'transform-numeric-separator',
```

Comment:
I did not check, but iirc there was a reason for those to be here so old browsers do not choke on those syntaxes. Maybe its no longer needed with Vite, but I remember that we had to add those explicitely to have babel transform those features.

Reply:
i checked the output, and it looks like Vite rewrites the numbers by default, so this:
```ts
const TEST = 100_000_000;
const TEST2 = 1_0;
```
becomes:
```js
const T=1e8;
const T2=10;
```

Reply:
Ok, can you check for the other ones as well? I remember that we had issues because the nullish coalescing operator (`??`) was not properly transpiled in some browsers.

But if this is no longer the case, then maybe it means we need to update our `browserlistrc` to include those browsers rather than manually picking things to transpile?

Reply:
that seems untouched. if we want to support this, we should update our `browserlistrc` as all the things mentioned in this config have been supported for a while:

* https://caniuse.com/mdn-javascript_operators_nullish_coalescing
* https://caniuse.com/mdn-javascript_operators_optional_chaining
* https://caniuse.com/mdn-javascript_classes_private_class_methods

unless we we want to support IE11 i think it's fine to drop these.

Reply:
Ok I looked back at the git history, and those were here because webpack or the minifier did not understood those syntaxes, so we needed to transpile them.

Now its all supported by Vite, so we can drop those plugins.